sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -658,14 +658,15 @@ class Analyzer(override val catalogManager: CatalogManager)
     // CUBE/ROLLUP/GROUPING SETS. This also replace grouping()/grouping_id() in resolved
     // Filter/Sort.
     def apply(plan: LogicalPlan): LogicalPlan = plan resolveOperatorsDown {
-      case h @ UnresolvedHaving(_, agg @ Aggregate(Seq(gs: GroupingSet), aggregateExpressions, _))
+      case h @ UnresolvedHaving(
+      _, agg @ Aggregate(Seq(gs: GroupingAnalytic), aggregateExpressions, _))
```

Comment:
nit: to avoid the line break, how about renaming `aggregateExpressions` -> `aggExprs`?

Reply:
> nit: to avoid the line break, how about renaming `aggregateExpressions` -> `aggExprs`?

Done